### DISCUSSION TITLE
Supabase && GatsbyJS anyone?

### DISCUSSION THREAD

**Discussion by klutchdev** [supabase/supabase#501] [Feature Requests]
Would anyone be interested in Supabase/GatsbyJS examples/sample code?

It is very similar to NextJS, but it never hurts to have both - the more, the merrier!!

I'd be more than happy to get a couple little apps locked and loaded for the JavaScript SDK examples folder
if anyone finds it useful.

Let me know!!!! 🔥

**Comment by kiwicopple** [comment]
It would be very cool to have a `gatsbyjs-todo-list` example! https://github.com/supabase/supabase/tree/master/examples

Then we can do a walkthrough of it in our Docs
